src/ConductorSharp.Engine/Model/DoWhileTaskModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using ConductorSharp.Engine.Builders;
+using MediatR;
+
+namespace ConductorSharp.Engine.Model
+{
+    /// <summary>
+    /// Input for configuration of the DO_WHILE task.
+    /// </summary>
+    public class DoWhileInput : IRequest<NoOutput>, IWorkflowInput
```

Comment:
I noticed input can contain value which is used to define loop condition. You should define some "Value" property here which consumer can use in loop condition